_posts/2016/2016-11-21-multitenancy-and-elasticsearch.md

```diff
@@ -0,0 +1,127 @@
+---
+layout: post
+title: "Multitenancy and Elasticsearch"
+date: "2016-11-21 09:00:00"
+image: blog-header/unsplash.jpg
```

Comment:
rename "unsplash" to "multitenant"

Is the picture from here? https://www.pexels.com/photo/woman-in-gray-knit-sweater-holding-files-26662/